src/report_reader.cpp

```diff
@@ -359,19 +355,21 @@ DataFrame<T> ReportReader<T>::Population::get(const Selection& selection,
             .select({index_start, it->second.first},
                     {index_stop - index_start + 1, it->second.second - it->second.first})
             .read(data);
+        int timer_index = 0;
+        for (const std::vector<float>& datum : data) {
+            for (float d : datum) {
+                data_frame.data[timer_index].push_back(d);
+            }
+            ++timer_index;
+        }
```

Comment:
Sorry I am lost again, in python I have : 
data_frame.data[x][y] with x == nodes and y == times

You have the opposite here. Is this due to the C vs F representation in bindings ?

Reply:
or  is  it `ids_index` instead ?

Reply:
I have no idea, it's  clearly difficult to tell, too many people read things as they want in the middle ^^

Reply:
In fact if we look at the c++ tests it's clearly [times][ids]

Reply:
Yes I checked again I messed up this is indeed [times][ids]